eclipse-theia/theia
# [cpp] remove '@theia/cpp' extension

#### What it does


Fixes #6340

The `@theia/cpp` extension is planned to move to [theia-cpp-extensions](https://github.com/eclipse-theia/theia-cpp-extensions/) repository.
The pull request has already been started: https://github.com/eclipse-theia/theia-cpp-extensions/pull/69

_This PR:_
- removes the `@theia/cpp` extension.
- removes references to the `@theia/cpp` extension.

#### How to test


- verify that CI passes successfully
- verify that the application correctly starts

#### Review checklist

- [x] `@theia/cpp` is merged into [theia-cpp-extensions](https://github.com/eclipse-theia/theia-cpp-extensions/)

#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-off-by: vince-fugnitto <vincent.fugnitto@ericsson.com>
